Modules/posixmodule.c

```diff
@@ -285,7 +285,7 @@ corresponding Unix manual entries for more information on calls.");
 #  undef HAVE_SCHED_SETAFFINITY
 #endif
 
-#if defined(HAVE_SYS_XATTR_H) && defined(__linux__) && !defined(__FreeBSD_kernel__) && !defined(__GNU__)
+#if defined(HAVE_SYS_XATTR_H) && defined(HAVE_LINUX_LIMITS_H) && defined(__linux__) && !defined(__FreeBSD_kernel__) && !defined(__GNU__)
```

Comment:
Should be enough with just the `HAVE_LINUX_LIMITS_H` guard, right?

```suggestion
#if defined(HAVE_SYS_XATTR_H) && defined(HAVE_LINUX_LIMITS_H) && !defined(__FreeBSD_kernel__) && !defined(__GNU__)
```

Reply:
Makes sense to me.